packages/react-native-reanimated/scripts/validate-apple.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-if grep -Rn apple -e '#include ' ; then
+if grep -Rn apple -e '^#include "'; then
```

Comment:
This should be just `#include ` so that we also catch `#include <foo.h>`
```suggestion
if grep -Rn apple -e '^#include '; then
```